bin/s2nc.c

```diff
@@ -98,6 +101,10 @@ void usage()
                     "    Ex: --psk psk_id,psk_secret,SHA256 --psk shared_id,shared_secret,SHA384.\n");
     fprintf(stderr, "  -E ,--early-data <file path>\n");
     fprintf(stderr, "    Sends data in file path as early data to the server. Early data will only be sent if s2nc receives a session ticket and resumes a session.");
+    fprintf(stderr, "  --perf-send-amount <bytes>\n");
```

Comment:
The last line of the early data documentation is missing a newline, so your option description gets combined with it. Can you append the missing newline?